drivers/wifi/simplelink/simplelink_sockets.c

```diff
@@ -518,7 +518,7 @@ static ssize_t simplelink_sendto(int sd, const void *buf, size_t len,
 /*
  * Later SimpleLink SDK versions implement the full getaddrinfo semantics,
  * returning potentially multiple IP addresses.
- * This version implements a simple gethostbyname() API.
+ * This version implements a simple gethostbyname() API for client only.
```

Comment:
Not sure about this change.

Reply:
Well, it's true it's not specifically required for the coverity bug fix, but some indication that the implementation is naive, just sufficient for its current usage in Zephyr.